sqlalchemy_filters/filters.py

```diff
@@ -29,6 +29,9 @@ class Operator(object):
     }
 
     def __init__(self, operator):
+        if operator is None:
```

Comment:
also testing the `operator` simply for true value would be enough, no need for explicit `None` test .)